pycbc/types/config.py

```diff
@@ -95,11 +96,24 @@ def __init__(
             overrideTuples = []
         if deleteTuples is None:
             deleteTuples = []
-        DeepCopyableConfigParser.__init__(self)
+
+        super().__init__()
 
         # Enable case sensitive options
         self.optionxform = str
 
+        # Add in environment
+        # We allow access to environment variables by adding them into the
```

Comment:
Is there necessarily an `os_env_vals` _section_ of the config file? 
I guess the syntax here effectively creates a section in configparse-speak, even though any actual file may not have one. 
So maybe phrase as `by loading them as a config section called os_env_vals.`

Reply:
This does add in a requirement that there is *not* another section present called `os_env_vals`. If (for e.g.) one tries to use an executable with this name it's going to be sent your entire environment as command-line arguments (in an earlier version of this branch, that happened for every executable, not a good thing to do!)

I guess I need to make this clearer that this is a "special" section (like `[workflow]` or `[executables]`), which has special meaning to pycbc.workflow (and indeed inference config files).